SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/SpatialDataDebugMap.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/SpatialDataDebugMap.h"
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPropertyReplication/SpatialTestPropertyReplicationMultiworker.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPropertyReplication/SpatialTestPropertyReplicationSubobject.h"
+#include "TestWorkerSettings.h"
+
+USpatialDataDebugMap::USpatialDataDebugMap()
+	: UGeneratedTestMap(EMapCategory::CI_PREMERGE_SPATIAL_ONLY, TEXT("SpatialDataDebugMap"))
+{
+	SetNumberOfClients(2);
+
+	// clang-format off
+	SetCustomConfig(TEXT("[/Script/EngineSettings.GeneralProjectSettings]") LINE_TERMINATOR
+					TEXT("bSpatialAuthorityDebugger=True"));
+	// clang-format on
+}
+
+void USpatialDataDebugMap::CreateCustomContentForMap()
+{
+	ULevel* CurrentLevel = World->GetCurrentLevel();
+
+	FTransform Server1Pos(FVector(250, -250, 0));
```

Comment:
All tests have this flaw, so nothing to change here, but its not obvious what these numbers mean :/